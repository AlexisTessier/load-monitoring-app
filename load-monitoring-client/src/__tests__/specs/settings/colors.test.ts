import {
  green,
  orange,
  red
} from '../../../settings/colors'

test('green', () => {
  expect(typeof green).toBe('string')
})

test('orange', () => {
  expect(typeof orange).toBe('string')
})

test('red', () => {
  expect(typeof red).toBe('string')
})

test('snapshot', () => {
  expect({
    green,
    orange,
    red
  }).toMatchSnapshot()
})
