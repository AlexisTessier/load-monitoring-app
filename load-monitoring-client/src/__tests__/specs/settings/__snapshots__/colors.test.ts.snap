// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Object {
  "green": "#d1db4a",
  "orange": "#ff7950",
  "red": "#f2615a",
}
`;
